Xamarin.Forms.Platform.Tizen/Extensions/GeometryExtensions.cs

```diff
@@ -0,0 +1,223 @@
+
+namespace Xamarin.Forms.Platform.Tizen
+{
+    public static class GeometryExtensions
+    {
+        public static SKPath ToSKPath(this Geometry geometry)
+        {
+            SKPath path = new SKPath();
+
+            if (geometry is LineGeometry)
+            {
+                path = MakePath(geometry as LineGeometry);
+            }
+            else if (geometry is RectangleGeometry)
+            {
+                path = MakePath(geometry as RectangleGeometry);
+            }
+            else if (geometry is EllipseGeometry)
+            {
+                path = MakePath(geometry as EllipseGeometry);
+            }
+            else if (geometry is GeometryGroup)
+            {
+                path = MakePath(geometry as GeometryGroup);
+            }
+            else if (geometry is PathGeometry)
+            {
+                path = MakePath(geometry as PathGeometry);
+            }
+
+            return path;
+        }
+
+        static SKPath MakePath(LineGeometry lineGeometry)
+        {
+            var path = new SKPath();
+            path.MoveTo(
+                Forms.ConvertToScaledPixel(lineGeometry.StartPoint.X),
+                Forms.ConvertToScaledPixel(lineGeometry.StartPoint.Y));
+
+            path.LineTo(
+                Forms.ConvertToScaledPixel(lineGeometry.EndPoint.X),
+                Forms.ConvertToScaledPixel(lineGeometry.EndPoint.Y));
+
+            return path;
+        }
+
+        static SKPath MakePath(RectangleGeometry rectangleGeometry)
+        {
+            var path = new SKPath();
+            FormsRectangle rect = rectangleGeometry.Rect;
+
+            path.AddRect(new SKRect(
+                Forms.ConvertToScaledPixel(rect.Left),
+                Forms.ConvertToScaledPixel(rect.Top),
+                Forms.ConvertToScaledPixel(rect.Right),
+                Forms.ConvertToScaledPixel(rect.Bottom)),
+                SKPathDirection.Clockwise);
+
+            return path;
+        }
+
+        static SKPath MakePath(EllipseGeometry ellipseGeometry)
+        {
+            var path = new SKPath();
+            path.AddOval(new SKRect(
+                    Forms.ConvertToScaledPixel(ellipseGeometry.Center.X - ellipseGeometry.RadiusX),
+                    Forms.ConvertToScaledPixel(ellipseGeometry.Center.Y - ellipseGeometry.RadiusY),
+                    Forms.ConvertToScaledPixel(ellipseGeometry.Center.X + ellipseGeometry.RadiusX),
+                    Forms.ConvertToScaledPixel(ellipseGeometry.Center.Y + ellipseGeometry.RadiusY)),
+                    SKPathDirection.Clockwise);
+
+            return path;
+        }
+
+        static SKPath MakePath(GeometryGroup geometryGroup)
+        {
+            var path = new SKPath();
+            path.FillType = geometryGroup.FillRule == FillRule.Nonzero ? SKPathFillType.Winding : SKPathFillType.EvenOdd;
+
+            foreach (Geometry child in geometryGroup.Children)
+            {
+                SKPath childPath = child.ToSKPath();
```

Comment:
How about using MakePath() method?
```suggestion
                SKPath childPath = MakePath(child);
```

Reply:
Done updating.